.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Excuse me, What does this mean?

Reply:
These are automatically generated by Husky